<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>525</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Курсач</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/images/images/icon.jpg</normaloff>:/new/images/images/icon.jpg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="carAdd">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Поезда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Купленные места</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Купе</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Плацкарт</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Тип места</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Кол-во мест и стоимость</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1001</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10001</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Маршрут поезда</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FAQ">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>741</x>
     <y>10</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ReSize">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-&gt;</string>
   </property>
  </widget>
  <zorder>tableView</zorder>
  <zorder>carAdd</zorder>
  <zorder>tableView_2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>groupBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_5</zorder>
  <zorder>FAQ</zorder>
  <zorder>ReSize</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
